swarm/api/act.go

```diff
@@ -383,7 +401,7 @@ func DoPKNew(ctx *cli.Context, privateKey *ecdsa.PrivateKey, granteePublicKey st
 	return sessionKey, ae, nil
 }
 
-func DoACTNew(ctx *cli.Context, privateKey *ecdsa.PrivateKey, salt []byte, grantees []string) (accessKey []byte, ae *AccessEntry, actManifest *Manifest, err error) {
+func DoACTNew(ctx *cli.Context, privateKey *ecdsa.PrivateKey, salt []byte, grantees []string, encryptPassword string) (accessKey []byte, ae *AccessEntry, actManifest *Manifest, err error) {
 	if len(grantees) == 0 {
```

Comment:
now that we allow passwords, we should not return if no grantees via pubkey. only if both are empty no?

Reply:
yes